src/app/screens/Calculator/index.js

```diff
@@ -56,7 +58,14 @@ const Calculator = () => {
     if (resultRef.current === '' && isValidRef.current) return;
     if (resultRef.current !== calcRef.current) {
       setDidCommit(true);
-      setCalc(isValidRef.current ? resultRef.current : SYN_ERR);
+      if (isValidRef.current) {
+        dispatch.dispatch(OperationActions.addOperation(`${calcRef.current} = ${resultRef.current}`), [
+          dispatch
+        ]);
```

Comment:
A few things over here.
First, extract this into a constant:
`${calcRef.current} = ${resultRef.current}`
Then, why are you doing `dispatch.dispatch` and at the end the `[dispatch]`?

Reply:
Fixed